zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/internal/HttpBulkIndexer.java

```diff
@@ -18,6 +18,7 @@
 import java.util.Iterator;
 import java.util.LinkedHashSet;
 import java.util.Set;
+import java.util.concurrent.RejectedExecutionException;
```

Comment:
can RejectedExecutionException be moved to a new pull request we can merge?

Reply:
See #2511.